services/user.py

```diff
@@ -0,0 +1,63 @@
+from typing import Optional
+
+from django.contrib.auth import get_user_model
+
+from db.models import User
```

Comment:
The import of `User` from `db.models` is unnecessary since `get_user_model()` is used for user operations. Consider removing this import to clean up the code.